application/models/Stripe_Model.php

```diff
@@ -38,61 +35,56 @@ function GenSessionId ($customer_email, $eventData) {
 
         ]);
         $stripeSession = array($session);
-        $sessID = ($stripeSession[0]['id']);
-        return $sessID;
+        return ($stripeSession[0]['id']);
     }
 
     /**
-	* This function checks if the stripe session has made it's payment successfully
-	*
-	* @param string    	$session_id  		The session id of this current payment session
-	*/
-    function CheckPayment($session_id)
+     * This function checks if the stripe session has made it's payment successfully.
+     *
+     * @param string $sessionId The session id of this current payment session.
+     *
+     * @return bool If the user has paid.
+     */
+    function checkPayment($sessionId)
     {
-        require_once('vendor/autoload.php');
-        //session_start();
-
         $hasPaid = False;
 
-        \Stripe\Stripe::setApiKey(SECRETKEY);
+        Stripe::setApiKey(STRIPE_PRIVATE_KEY);
 
-        $events = \Stripe\Event::all([
-        'type' => 'checkout.session.completed',
-        'created' => [
-            // Check for events created in the last 3 minutes
-            'gte' => time() - 1 * 10 * 60,
-        ],
+        $events = Event::all([
+            'type' => 'checkout.session.completed',
+            'created' => [
+                // Check for events created in the last 3 minutes
+                'gte' => time() - 1 * 10 * 60,
+            ],
         ]);
 
         foreach ($events->autoPagingIterator() as $event) {
-            //getting each session object
             $session = $event->data->object;
 
-            //Checking if a paid session id matches with current session id
-            if ($session->id == $session_id) {
+            // Check if a paid session ID matches with current session ID
+            if ($session->id == $sessionId) {
                 $hasPaid = True;
                 break;
             }
         }
-        return  $hasPaid;
+
+        return $hasPaid;
     }
 
     /**
-     * This Function returns the customers email given a session id
+     * Returns the customers email given a session ID.
      *
-     * @param string $session_id             The session id of the desired email address
+     * @param string $sessionId The session id of the desired email address.
      *
-     * @return $email the email address of a given session
+     * @return string $email the email address of a given session.
      */
-    function GetEmail($session_id) {
-
-        \Stripe\Stripe::setApiKey(SECRETKEY);
+    function getEmail($sessionId)
+    {
+        Stripe::setApiKey(STRIPE_PRIVATE_KEY);
```

Comment:
So by Ctrl+LeftClick on this one it now takes you to the constants file aye? So the usage is the same, that's nice!